yeat/bandage/bandage.py

```diff
@@ -0,0 +1,38 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2022, DHS. This file is part of YEAT: http://github.com/bioforensics/yeat
+#
+# This software was prepared for the Department of Homeland Security (DHS) by the Battelle National
+# Biodefense Institute, LLC (BNBI) as part of contract HSHQDC-15-C-00064 to manage and operate the
+# National Biodefense Analysis and Countermeasures Center (NBACC), a Federally Funded Research and
+# Development Center.
+# -------------------------------------------------------------------------------------------------
+
+from pkg_resources import resource_filename
+from snakemake import snakemake
+import subprocess
+import warnings
+
+
+def check_bandage():
+    completed_process = subprocess.run(["Bandage", "--help"], capture_output=True, text=True)
```

Comment:
I am running ["Bandage", "--help"] instead of just ["Bandage"] for a reason.

When running just ["Bandage"], the command will open up the GUI and the GUI will put a pause/hold in the terminal. As a result, the snakemake workflow pauses until the user kills the GUI.

By doing ["Bandage", "--help"], we get the Bandage CLI and the help info.

Overall, I did this command because this command will tell YEAT and the user if Bandage is installed or not.

If it is not installed, the user will find this error message saying that "Bandage is not a directory!"